### PR TITLE
rename folder without skeleton

### PR DESCRIPTION
**Pull request by haribhandari07**
## Description

Do not use skeleton files for ```webUIRenameFolders``` to speed up CI
## Related Issue

- Fixes <issue_link>
Part of https://github.com/owncloud/QA/issues/622
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/features/webUIRenameFolders/renameFolders.feature [hunk 1]
```diff
@@ -5,33 +5,37 @@ Feature: rename folders
   So that I can organise my data structure
 
   Background:
-    Given user "user1" has been created with default attributes and skeleton files
-    And user "user1" has logged in using the webUI
```
Comment (individual-it) [inline tests/acceptance/features/webUIRenameFolders/renameFolders.feature]: Why deleting the login and browse step from the background?

**Review by individual-it — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/webUIRenameFolders/renameFolders.feature (hunk 1 above)
Comment (dpakach) [reply tests/acceptance/features/webUIRenameFolders/renameFolders.feature]: If we log in at beginning then we would have to reload the page after creating the files if we want to interact with those files through the webUI. IMO that just makes the tests a bit slower.

**Review by dpakach — COMMENTED** [review]
LGTM :+1:

**Review by phil-davis — APPROVED** [review]

**Review by individual-it — APPROVED** [review]

**Comment by HariBhandari07** [comment]
Backport on #35496

**Pull request merged**